### ISSUE TITLE
A11y_30days_Welcome to 30 Days of Learning_High Contrast: Github icon is not visible in high contrast desert theme 

### ISSUE THREAD

**Issue by Ashutoshsahoo164** [microsoft/30daysof#176]
"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### GitHub Tags:
#A11yTCS;#A11yMAS;#A11ySev2;#30days;#P3_30days_Dec22;#WCAG4.3.3;#DesktopWeb;#Win11;#ChromiumEdge

### Environment Details:
Application Name: 30days 
#URL: https://microsoft.github.io/30daysof/
Windows Version: Win11
Chromium Edge Version: 110.0.1556.0 (Official build) dev (64-bit)
### Pre-Requisite:
To apply high contrast, follow the below steps:
1.	Open settings, tab till Accessibility and press enter
2.	Tab till Contrast themes and press enter
3.	Tab to the contrast theme combo box and press enter. Navigate to Desert theme using down arrow key
4.	Tab to apply and press enter

### Repro Steps:
1.	Hit the URL: https://microsoft.github.io/30daysof/ 
2.	Home page will open then tab to "Github" icon present in header section.
3.	Observe “Github” icon is not visible in high contrast desert theme.
### Actual:
Github icon is not visible in high contrast desert theme
### Expected:
Github icon should be visible properly in high contrast desert theme
### User Impact:
Low vision user will not be able to understand the icon.
### Attachment:
![Github icon is not visible in high contrast desert theme](https://user-images.githubusercontent.com/85863895/209835974-a189696c-31f1-40b1-b3ae-964d5f36e110.gif)

**Comment by Raghu-addepalli** [comment]
Verified the issue in below environment issue is fixed, hence closing the bug.
org - https://microsoft.github.io/30daysof/
![176_Closed](https://user-images.githubusercontent.com/91586743/219577299-58da2b5e-b8e0-448f-8483-6e3b8eae4f89.png)
Git Hub Tags: #Closed;

**Comment by Ashutoshsahoo164** [comment]
GithubTags: #Closed;
